src/functionHandlers/notarisePdt/validateInputs/validateDocument.ts

```diff
@@ -17,6 +19,17 @@ import {
   DocumentInvalidError,
 } from "../../../common/error";
 import { config } from "../../../config";
+import clinicProviderSchema from "../../../static/clinic-provider-schema.json";
```

Comment:
Mm can we do without this? Want to avoid having the schema in separate places.

Maybe we can axios.get directly from `https://schemata.openattestation.com/sg/gov/moh/pdt-healthcert/2.0/clinic-provider-schema.json` to obtain it (just like the how the other schemas will be fetched with the `loadSchema()` function).

Reply:
Another way is to obtain it from `@govtechsg/oa-schemata` package as well. But not sure if it currently exports the schema 🤔

Reply:
yea.. i'm also look in that oa schemata package. there is no export json from there. let me try to load from http url

Reply:
ok.. i'm just change it with axios call but there need to do some trick like this https://github.com/Notarise-gov-sg/api-notarise-healthcerts/pull/503/files#diff-c572ec439dc05bfdd435f9fbd6a9116e24b5a87ccaac0025b6372a33a44d8d1fR73

Reply:
Sorry Zaw, I just realised a downside to this approach. This means that in every API request, we will be unnecessarily making multiple fetch API calls which could slow down our response (or fail if `schemata.openattestation.com` is ever unreachable).

I think we have to load all the schemas by file for better performance (i.e. without using `loadSchema()`):
- **Main Schema**: https://schemata.openattestation.com/sg/gov/moh/pdt-healthcert/2.0/clinic-provider-schema.json
- https://schema.openattestation.com/2.0/schema.json
- https://schemata.openattestation.com/sg/gov/moh/pdt-healthcert/2.0/schema.json
- https://schemata.openattestation.com/sg/gov/moh/fhir/4.0.1/lite-schema.json
- (Not sure if I miss anything else)

Reply:
hmm... right. load this as file is better. i think only this . let revise it and validate it again.